### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Mu7ammadAbed** [GSG-G10/news-website-team-5#22]
Lina gave you a tool to test performance and accessibility you should use it and `input` tag should have a `label` for accessibility purposes and if you don't want the label text to be printed on the screen you can use the attributes like `aria-label` or `title`
